HW_1/task_1.py

```diff
@@ -0,0 +1,29 @@
+# Была выполнена 1 задача из домашнего задания номер 1.
+
+"""
+Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. 
+Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны с суммой двух других. 
+Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника с такими сторонами не существует. 
+Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним. 
+"""
+
+SIDE_A = 2
+SIDE_B = 2
+SIDE_C = 2
+
+if SIDE_A >= (SIDE_B + SIDE_C):
```

Comment:
Здесь и ниже скобки не нужны
```suggestion
if SIDE_A >= SIDE_B + SIDE_C:
```